### PR TITLE
Con 3306 20230227 add command line args url apikey

### PR DESCRIPTION
**Pull request by nbutler77**
Extends CVC to allow users to pass a Conservator URL and API key.
This will allow Conservator to call Conservator-CLI for an arbitrary user.

**File Changes:**

`FLIR/conservator/cli/cvc.py`
 - Added url, api key parameters to `main`, `clone`

`FLIR/conservator/config.py`
`FLIR/conservator/connection.py`
 - Cleaned up some pylint issues

`FLIR/conservator/conservator.py`
 - Added static `from_config_dict` function that will create a Conservator instance using a config created from a supplied dict

`docs/usage/cvc_guide.rst`
 - Added note about using cvc with URL, API key parameters

[[JIRA Task](https://jiracommercial.flir.com/browse/CON-3306)]

### CODE REVIEW

File: FLIR/conservator/connection.py [hunk 1]
```diff
@@ -31,8 +31,6 @@ class ConservatorMalformedQueryException(Exception):
     fault.
     """
 
-    pass
-
```
Comment (rlc2) [inline FLIR/conservator/connection.py:35]: The class definition is empty, shouldn't the pass statement be left here?

File: FLIR/conservator/conservator.py [hunk 2]
```diff
@@ -134,5 +153,3 @@ class UnknownMediaIdException(Exception):
     """
     Raised when a media ID cannot be resolved to a Video or Image.
     """
-
-    pass
```
Comment (rlc2) [inline FLIR/conservator/conservator.py:138]: Removed another `pass` statement that closes an empty class definition.

File: FLIR/conservator/connection.py (hunk 1 above)
Comment (nbutler77) [reply FLIR/conservator/connection.py:35]: PyLint seemed to think it was unnecessary, but I'll put both pass statements back in.

**Review by rlc2 — APPROVED** [review]

**Comment by SJLC** [comment]
@nbutler77 there was a merge conflict so I pushed a merge-from-main commit to resolve it

**Review by SJLC — APPROVED** [review]

**Pull request merged**